orm/index.go

```diff
@@ -90,11 +120,14 @@ func (i Index) Update(db weave.KVStore, prev Object, save Object) error {
 // GetLike calculates the index for the given pattern, and
 // returns a list of all pk that match (may be empty), or an error
 func (i Index) GetLike(db weave.ReadOnlyKVStore, pattern Object) ([][]byte, error) {
-	index, err := i.index(pattern)
+	indexes, err := i.index(pattern)
```

Comment:
Yeah, I am not sure of this pattern when there is multi-index...
Maybe we can remove it if not heavily used....